src/controllers/user.js

```diff
@@ -10,9 +11,10 @@ export const create = async (req, res) => {
     if (existingUser) {
       return sendDataResponse(res, 400, { email: 'Email already in use' })
     }
-
+    if (!validator.validate(userToCreate.email)) {
```

Comment:
@alsyoufomar has PR open that introduces a general purpose validation library - it's OK to have both of these for a time, but you guys should meet to discuss if we can use that library for email validation too and update this in another pull request at some point next week. Ideally, if possible, we'd all use the same library to do validation - it might not be possible, these might serve different purposes, but it's worth finding out.

Reply:
yeah, we'd had quick chat already yesterday, about him doing validation that covers more ground than just email. Just got to sit down and go over with him what the libary is & how to apply

Reply:
Also - fantastic to see you guys having these cross team conversations - great stuff! Some real world teams don't work that well ;)